ansible/roles/nginx/vars/tableau.yml

```diff
@@ -13,7 +13,7 @@ nginx_sites:
    - "X-Forwarded-Host $host:443"
    location1:
     name: /
-    proxy_pass: http://tableau1.internal.commcarehq.org
+    proxy_pass: http://tableau3.internal.commcarehq.org
```

Comment:
I feel like there's a world where this lives in the config file, and that that world is a slightly better world. Maybe next time you have to move the server or reconfigure another one on a different setup, you can think about how to enter that world.